admin_guide/pruning_resources.adoc

```diff
@@ -645,3 +645,29 @@ finished, the registry can be switched back to read-write mode by executing:
 $ oc env -n default dc/docker-registry REGISTRY_STORAGE_MAINTENANCE_READONLY-
 ----
 endif::[]
+
+[[pruning-cronjobs]]
+== Pruning Cron Jobs
+
+[IMPORTANT]
+====
+Cron Jobs is a Technology Preview feature only.
+ifdef::openshift-enterprise[]
+Technology Preview features are not
+supported with Red Hat production service level agreements (SLAs), might not be
+functionally complete, and Red Hat does not recommend to use them for
+production. These features provide early access to upcoming product features,
+enabling customers to test functionality and provide feedback during the
+development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+At this time, the results of cron jobs are not automatically pruned, unless a user
+explicitly configure that feature. Therefore, cluster administrator should perform regular
```

Comment:
s/configure/configures

And, I think from context, I think it should be s/user/application developer